src/utils.ts

```diff
@@ -282,17 +282,25 @@ export function areObjectsEqual(
  * @param b the regExp to test agains
  * @returns are the regExps equal
  */
-export function areRegExpsEqual(a: RegExp, b: RegExp) {
-  return (
-    a.source === b.source &&
-    a.global === b.global &&
-    a.ignoreCase === b.ignoreCase &&
-    a.multiline === b.multiline &&
-    a.unicode === b.unicode &&
-    a.sticky === b.sticky &&
-    a.lastIndex === b.lastIndex
-  );
-}
+export const areRegExpsEqual = (() => {
+  if (/foo/g.flags === 'g') {
+    return function areRegExpsEqual(a: RegExp, b: RegExp) {
+      return a.source === b.source && a.flags === b.flags;
```

Comment:
Could the flags be in a different order? 'gm' vs 'mg'

Reply:
Based on the [description in MDN](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/RegExp/flags), which is based on the spec, the flags are ordered alphabetically. Are you seeing different results?

Reply:
Nope, I was just wondering. Sorry for the noise

Reply:
No worries!